OpenSim/Sandbox/xsens/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.1.3)
```

Comment:
Remove this line (and the CXX_STANDARD variables); they should be unnecessary. Also, this line would cause OpenSim to require cmake 3.1.3, and we have not yet made a decision on this (PR #925).